apps/tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,27 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2019 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "CANBus.h"
+#include "CANMsg.h"
+#include "mbed.h"
+#include "TutorialServo.h"
+
+// Blinking rate in milliseconds
+// #define BLINKING_RATE 500ms
+// AnalogIn potVoltageIn(PA_0);
+// PwmOut servoPwmOut(PA_1);
+
+CAN can(CAN1_RX, CAN1_TX);
```

Comment:
Not sure if it's required for the training, but FW uses the `CANBus` wrapper class instead of the mbed `CAN` API directly. See the arm app for example